huaweicloud/services/vpc/data_source_huaweicloud_networking_secgroups.go

```diff
@@ -28,6 +28,10 @@ func DataSourceNetworkingSecGroups() *schema.Resource {
 				Type:     schema.TypeString,
 				Optional: true,
 			},
+			"id": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
```

Comment:
missing behavior: `computed`, it is the default bahavior for `id`.

Reply:
done